language-server/src/server.js

```diff
@@ -142,14 +142,20 @@ documents.onDidChangeContent(async ({ document }) => {
 
   if (isSchema(document.uri)) {
     await waitUntil(() => isWorkspaceLoaded);
-    await validateSchema(document);
+    if (document.getText() !== "") {
+      await validateSchema(document);
+    }
   }
 });
 
 const validateSchema = async (document) => {
   const diagnostics = [];
 
   const instance = JsoncInstance.fromTextDocument(document);
+  if (instance === undefined) {
```

Comment:
Since `instance` should be a JsoncInstance representing `undefined`, you can use the `typeOf` function in JsoncInstance to determine if the it's representing an `undefined` value.